.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/app-prototype
+    docker:
+      - image: circleci/ruby:2.4.2-node-browsers
+        environment:
+          PGHOST: 127.0.0.1
+          PGUSER: postgres
+          RAILS_ENV: test
+      - image: circleci/postgres:9.6.5-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: app_prototype_test
+          POSTGRES_PASSWORD: ""
+    steps:
+      - checkout
+
+      # Restore bundle cache
+      - type: cache-restore
+        key: bundle-{{ arch }}
+
+      # Bundle install dependencies
+      - run: bundle install --path vendor/bundle --jobs=4
+      - run: bundle clean --force
+
+      # Store bundle cache
+      - type: cache-save
+        key: bundle-{{ arch }}
+        paths:
+          - vendor/bundle
+
+      # Database setup
+      - run: bundle exec rake db:create
+      - run: bundle exec rake db:schema:load
+
+      # Run rspec in parallel
+      - type: shell
+        command: |
+          bundle exec rspec --format RspecJunitFormatter \
+                            --out /tmp/test-results/rspec.xml \
+                            $(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)
+
+      # Save artifacts
+      - type: store_test_results
+        path: /tmp/test-results
+
+      # Ensure sample_data and seeds execute
+      - run: bundle exec rake db:seed db:sample_data
+
+      # Rubocop compliance
+      - run: bundle exec rubocop
```

Comment:
On our project we run fast tasks like linters first (i.e. before RSpec) so that the build can fail faster. It can be a bummer to wait 10 minutes for specs to run before finding out that your build is destined to fail because of a linting mistake.

Reply:
I like it.